src/components/AudioPlayer/Player/Player.container.js

```diff
@@ -2,31 +2,68 @@ import React from 'react';
 import propTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 
-import { setRandom } from 'store/actions/playerActions';
+import {
+  setIsPlaying,
+  setRandom,
+  setRepeat,
+  playPreviousSong,
+  playNextSong,
+} from 'store/actions/playerActions';
 import PlayerComponent from './Player.component';
 
 const PlayerContainer = ({ audioRef }) => {
   const songsQueue = useSelector((state) => state.songsQueue.get('queue'));
+  const isPlaying = useSelector((state) => state.player.get('isPlaying'));
   const isRandomEnabled = useSelector((state) => state.player.get('isRandomEnabled'));
+  const isRepeatEnabled = useSelector((state) => state.player.get('isRepeatEnabled'));
```

Comment:
I think I suggested the store to be named: `isRepeat` while in the selector name it `isRepeatEnabled`.

Reply:
I don't think so, but i'll change it 😄